apps/dashboard/components/logs/checkbox/filters-popover.tsx

```diff
@@ -17,15 +27,19 @@ type FiltersPopoverProps = {
   items: FilterItemConfig[];
   activeFilters: FilterValue[];
   getFilterCount?: (field: string) => number;
+  open?: boolean;
+  onOpenChange?: Dispatch<SetStateAction<boolean>>;
 };
 
 export const FiltersPopover = ({
   children,
   items,
   activeFilters,
+  open,
+  onOpenChange,
   getFilterCount = (field) => activeFilters.filter((f) => f.field === field).length,
 }: PropsWithChildren<FiltersPopoverProps>) => {
-  const [open, setOpen] = useState(false);
+  const isMobile = useMediaQuery("(max-width: 768px)");
```

Comment:
having these magic values is a footgun I think
we'd need to define them in every component, that we make mobile ready, and it'll become a mess.

if we want to use media queries, we should have that hook (and the pixel value) defined in a central place